bolt_windows.go

```diff
@@ -49,27 +49,23 @@ func fdatasync(db *DB) error {
 
 // flock acquires an advisory lock on a file descriptor.
 func flock(db *DB, mode os.FileMode, exclusive bool, timeout time.Duration) error {
```

Comment:
`mode` is probably not used anymore.

Reply:
Changed to `_`. Has to be in parameters to match other platform implementation though.

Reply:
I don't think it's used in any of the other platforms. It was just there for Windows.

Reply:
Ah, indeed. Removed.